dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/internal/AccessTokenCacheHelper.java

```diff
@@ -72,21 +75,35 @@ boolean isTokenInCachedResultExpired(ProviderAuthenticationResult cachedResult,
 
     @FFDCIgnore(Exception.class)
     @SuppressWarnings("unchecked")
-    long getTokenExpirationFromCustomProperties(Hashtable<String, Object> customProperties) {
+    boolean doesPropertyExistInAccessTokenInfo(String propertyName, Hashtable<String, Object> customProperties) {
         if (customProperties == null || customProperties.isEmpty()) {
```

Comment:
Since the caller isTokenInCachedResultExpired already checks for null or empty, this block will never execute.

Reply:
There is no need to remove this block at this moment.